src/core/ext/xds/xds_server_config_fetcher.cc

```diff
@@ -292,7 +297,8 @@ class XdsServerConfigFetcher::ListenerWatcher::FilterChainMatchManager::
         filter_chain_match_manager_(std::move(filter_chain_match_manager)) {}
 
   void OnResourceChanged(
-      std::shared_ptr<const XdsRouteConfigResource> route_config) override {
+      std::shared_ptr<const XdsRouteConfigResource> route_config,
+      RefCountedPtr<ReadDelayHandle> /* read_delay_handle */) override {
```

Comment:
If the parameter is not given a name, does it actually stay in scope until the function returns?  I'm not sure, but we should verify.